### ISSUE TITLE
Expose trigger areas

### ISSUE THREAD

**Issue by FredrikNoren** [AmbientRun/Ambient#793]
It's currently not possible to create trigger areas, but we do support it in the engine. Might be as easy as just exposing the `collider_type` component.

**Comment by philpax** [comment]
How would this work as a user? You create a physics object with some shape, set it to a TriggerArea collider type, and then when you get a Collision message you check if it's that TriggerArea? (Just confirming my understanding)

Also trying to figure out if this overlaps with #281.

**Comment by FredrikNoren** [comment]
@philpax Yeah exactly. I think #281 is the more general, better long term solution, but to at least make it possible to create trigger areas right now I think we could just expose this component.
